src/components/cc-ssh-key-list/cc-ssh-key-list.js

```diff
@@ -31,6 +31,8 @@ const SKELETON_KEYS = [{
 /**
  * @typedef {import('./cc-ssh-key-list.types.js').CreateSshKeyFormState} CreateSshKeyFormState
  * @typedef {import('./cc-ssh-key-list.types.js').KeyDataState} KeyDataState
+ * @typedef {import('./cc-ssh-key-list.types.js').NewKey} NewKey
+ * @typedef {import('./cc-ssh-key-list.types.js').SshKey} SshKey
  */
```

Comment:
**fix:** the `cc-ssh-key-list.types.d.ts` file does not export anything.
It should export the types listed here.